service/controller/resource/monitoring/remotewriteapiendpointconfigsecret/resource.go

```diff
@@ -59,7 +63,7 @@ func getObjectMeta(ctx context.Context, v interface{}) (metav1.ObjectMeta, error
 
 	return metav1.ObjectMeta{
 		Name:      key.RemoteWriteAgentConfigSecretName,
-		Namespace: key.Namespace(cluster),
+		Namespace: key.ClusterID(cluster),
```

Comment:
We cannot create them here now because CAPA clusters do not create a custom namespace but use the organization one

Reply:
See on golem

```
k get clusters -A
NAMESPACE        NAME    PHASE         AGE     VERSION
org-giantswarm   golem   Provisioned   43d     
org-giantswarm   vac07   Provisioned   4d11h 
```

Reply:
key.OrganizationNamespace should do the trick, but we will need to prefix the secret with the clusterID

Reply:
Oh actually, key.OrganizationNamespace should be used only for capi clusters but vintage should use ClusterID

Reply:
Yes, And how we can differentiate